vars_validation/validate_vars.py

```
@@ -0,0 +1,290 @@
+
+#!/usr/bin/env python3
```

Ruff (0.12.2)

Avoid whitespace before shebang

Remove whitespace before shebang

(EXE004)